plugin/metricstore/prometheus/options.go

```diff
@@ -51,10 +51,11 @@ func DefaultConfig() config.Configuration {
 		ServerURL:      defaultServerURL,
 		ConnectTimeout: defaultConnectTimeout,
 
-		MetricNamespace:   defaultMetricNamespace,
-		LatencyUnit:       defaultLatencyUnit,
-		NormalizeCalls:    defaultNormalizeCalls,
-		NormalizeDuration: defaultNormalizeCalls,
+		MetricNamespace:      defaultMetricNamespace,
+		LatencyUnit:          defaultLatencyUnit,
+		NormalizeCalls:       defaultNormalizeCalls,
+		NormalizeDuration:    defaultNormalizeCalls,
+		AdditionalParameters: nil,
```

Comment:
nil is the default value, no effect

Reply:
I see. Thanks. Removed it, as nil should be the default value.